xbanish.c

```diff
@@ -109,6 +111,10 @@ main(int argc, char *argv[])
 					ignored |= mods[i].mask;
 
 			break;
+        case 'k':
+            // choose which keyboard device to listen only (in case of xremap or Kmonad being used)
+            master_keyboard_device = strdup(optarg);
```

Comment:
`strdup` can fail since it allocates memory and so the return value needs to be checked.

But in this case, there should be zero reason to duplicate the string to begin with.

Reply:
thx, should be fixed